game/selenium_tests/test_boards.py

```
@@ -0,0 +1,165 @@
+        """Board has 64 squares and initial pieces rendered."""
+        log_info("Testing board render...")
+        self._start_pvp_game()
+
+        board = self.wait.until(
+            EC.presence_of_element_located((By.ID, 'board')),
+            message="Board not found"
+        )
+
+        squares = board.find_elements(By.CLASS_NAME, 'square')
+        self.assertEqual(len(squares), 64, f"Expected 64 squares, got {len(squares)}")
+        log_ok(f"64 squares found")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)